wfe/wfe.go

```diff
@@ -1365,23 +1354,17 @@ func (wfe *WebFrontEndImpl) Order(
 	request *http.Request) {
 
 	var account *core.Account
```

Comment:
I think you can delete this explicit `var` & the `account = acct` assignment on L1367 by changing L1362 to `account, prob := wfe.validPOSTAsGET(postData)`.